### PR TITLE
LATCH-905 Add new brand style for Telefonica

### PR DESCRIPTION
**Pull request by jaranaz**
## 🎟️ **Jira ticket**

https://jira.tid.es/browse/LATCH-905

## 🥅 **What's the goal?**

Latch app is applying the new brand style for Telefonica.
Thus, we need to add a new brand `.telefonica` to our list of available brands in Mistica iOS library.

## 🚧 **How do we do it?**

- Add the new `.telefonica` brand to Mistica iOS Makefile.
- Execute the script and generate the needed tokens for this new brand.
- Generate new snapshots for this new brand UI tests.

## 🏑 **AppCenter build**

qrcode (1)

### CODE REVIEW

**Comment by yceballost** [comment]
can you release a new version of mistica catalog to test it? thanks!

File: Tests/MisticaTests/UI/__Snapshots__/ButtonTests/testRegularSizeWithSecondaryStyle.with-telefonica-style.png
Comment (DavidMarinCalleja) [inline Tests/MisticaTests/UI/__Snapshots__/ButtonTests/testRegularSizeWithSecondaryStyle.with-telefonica-style.png:1]: @jaranaz in some button screenshots there are two vertical lines but in other screenshots (https://github.com/Telefonica/mistica-ios/pull/306/files#diff-9e507c290ed32dc6eff6720816cef263f1e7f9297d99f06c2d0d86fb0d12e7a1) those lines don't appear.

File: Tests/MisticaTests/UI/__Snapshots__/EmptyStatesTests/testBrandStyles.with-telefonica-dark-style.png
Comment (DavidMarinCalleja) [inline Tests/MisticaTests/UI/__Snapshots__/EmptyStatesTests/testBrandStyles.with-telefonica-dark-style.png:1]: Is this green square, right?

File: Tests/MisticaTests/UI/__Snapshots__/EmptyStatesTests/testBrandStyles.with-telefonica-style.png
Comment (DavidMarinCalleja) [inline Tests/MisticaTests/UI/__Snapshots__/EmptyStatesTests/testBrandStyles.with-telefonica-style.png:1]: Is this green square, right?

**Review by DavidMarinCalleja — APPROVED** [review]
**Good job!!**

File: Tests/MisticaTests/UI/__Snapshots__/ButtonTests/testRegularSizeWithSecondaryStyle.with-telefonica-style.png
Comment (jaranaz) [reply Tests/MisticaTests/UI/__Snapshots__/ButtonTests/testRegularSizeWithSecondaryStyle.with-telefonica-style.png:1]: You're right! I see is also happening with .viewNew brand

I talked to @WanaldinoTelefonica and it's a known issue with the snapshot library. Still pending to be resolved.

Comment (DavidMarinCalleja) [reply Tests/MisticaTests/UI/__Snapshots__/ButtonTests/testRegularSizeWithSecondaryStyle.with-telefonica-style.png:1]: Thanks, @jaranaz for checked it

**Review by idenjoe — APPROVED** [review]

**Comment by jaranaz** [comment]
@yceballost the [release](https://install.appcenter.ms/orgs/Tuenti-Organization/apps/Mistica-SwiftUI-iOS/distribution_groups/public/releases/37) of Mistica Catalog is ready to test with this new brand ;)

**Review by yceballost — APPROVED** [review]
nice!!

**Pull request merged**